Update How_to_retrieve_protocol_pending_calls.md (#2513)
SkylineCommunications/dataminer-docs: develop/devguide/Connector/Howto/How_to_retrieve_protocol_pending_calls.md

>
> - When the logging levels of the element are set to Level 2, some additional information will be available.
> - If you use the *Protocol Pendingcalls* option for an element that is stuck, you will not get any diagnostics info, because this feature will try to add a lock on the element and that will fail, since it is stuck. If you use the *Protocol Pendingcalls[no lock]* option instead, you will get some diagnostics info.

> > - In case of element issues while taking a [Log Collector package](xref:Collecting_data_to_report_an_issue_to_TechSupport), the tool will request the pending calls as well. This will be found as a subfolder within the logging folder.

Reviewer: I've tried to make the sentence a bit more clear below – have I interpreted everything correctly?

```suggestion
> - In case element issues are encountered while a [Log Collector package](xref:Collecting_data_to_report_an_issue_to_TechSupport) is generated, SLLogCollector will request the pending calls as well. You can find these in a subfolder within the logging folder.
```

Author: This is good :)
